### PR TITLE
[HOLD] Groundwork for AB testing with Adobe Target

### PR DESCRIPTION
**Pull request by wesbos**
// To run - this code would go inside Adobe Target
// You can open your console and run it to try it out

```js
$(function() {
  app.abTest.swap('heros/hero2', '.homepage-main');
});
```

### CODE REVIEW

File: heros/hero1.html.slim [hunk 1]
```diff
@@ -0,0 +1 @@
+= partial('hero1.html.slim');
```
Comment (paulrobinson) [inline heros/hero1.html.slim]: Why do you need this indirection? Can't you just put the contents of the partial inline?

Comment (paulrobinson) [inline heros/hero1.html.slim]: OK, I see now. I think you need the partial for the initial HTML build of the page, and then you use this file in the swap function. Is that right?

Comment (wesbos) [inline heros/hero1.html.slim]: Yep - we need the partials to be publicly available so we can pull them in
via the js if we are doing ab testing

**Comment by paulrobinson** [comment]
Closing this PR, as we've confirmed that this is the right approach. Please include this code in your PR containing the real experiment. E.i https://issues.jboss.org/browse/DEVELOPER-2763 and https://issues.jboss.org/browse/DEVELOPER-2779

**Pull request closed without merging**